database/schema/V1.2__schema.sql

```diff
@@ -0,0 +1,41 @@
+DROP VIEW application.view_find_sites;
```

Comment:
We will want to collapse all of the schema changes prior to a first release of code